openapi/paths/aml-checks@{id}.yaml

```
@@ -0,0 +1,26 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve a customer's AML check
```

```suggestion
  summary: Retrieve an AML check
```
I didn't see customerId in the path or query params here so would it be okay without the word customer?

---

Yeah, makes sense still, to me. I will change it.